plotly/figure_factory/_bullet.py

```diff
@@ -0,0 +1,342 @@
+            xaxis='x{}'.format(row + 1),
+            yaxis='y{}'.format(row + 1)
+        )
+
+        for k in scatter_options:
+            if k != 'marker':
+                markers[k] = scatter_options[k]
+
+        fig['data'].append(markers)
+
+        # titles and subtitles
+        title = df.iloc[row]['titles']
+        if 'subtitles' in df:
+            subtitle = '<br>{}'.format(df.iloc[row]['subtitles'])
+        else:
+            subtitle = ''
+        label = '<b>{}</b>'.format(title) + subtitle
+        annot = utils.annotation_dict_for_label(
+            label,
+            (num_of_lanes - row if orientation == 'h' else row + 1),
+            num_of_lanes,
+            vertical_spacing if orientation == 'h' else horizontal_spacing,
+            'row' if orientation == 'h' else 'col',
+            True if orientation == 'h' else False,
+            False
+        )
+        fig['layout']['annotations'].append(annot)
+
+    return fig
+
+
+def create_bullet(data, markers=None, measures=None, ranges=None,
+                  subtitles=None, titles=None, orientation='h',
+                  range_colors=None, measure_colors=None,
+                  horizontal_spacing=None, vertical_spacing=None,
+                  scatter_options={}, **layout_options):
+    """
+    Returns figure for bullet chart.
+
+    :param (pd.DataFrame | list) data: either a JSON list of dicts or a pandas
+        DataFrame.
+    :param (str) markers: the column name or dictionary key for the markers in
+        each subplot.
+    :param (str) measures: the column name or dictionary key for the measure
+        bars in each subplot. This bar usually represents the quantitative
+        measure of performance, usually a list of two values [a, b] and are
+        the blue bars in the foreground of each subplot by default.
+    :param (str) ranges: the column name or dictionary key for the qualitative
+        ranges of performance, usually a 3-item list [bad, okay, good]. They
+        correspond to the grey bars in the background of each chart.
+    :param (str) subtitles: the column name or dictionary key for the subtitle
+        of each subplot chart. The subplots are displayed right underneath
+        each title.
+    :param (str) titles: the column name or dictionary key for the main label
+        of each subplot chart.
+    :param (bool) orientation: if 'h', the bars are placed horizontally as
+        rows. If 'v' the bars are placed vertically in the chart.
+    :param (int) marker_size: sets the size of the markers in the chart.
+    :param (str | int) marker_symbol: the symbol of the markers in the chart.
+        Default='diamond-tall'
+    :param (list) range_colors: a list of two colors between which all
+        the rectangles for the range are drawn. These rectangles are meant to
+        be qualitative indicators against which the marker and measure bars
+        are compared.
+        Default=['rgb(198, 198, 198)', 'rgb(248, 248, 248)']
```

Comment:
I don't see `rgb(248, 248, 248)` anywhere else in the code.

Would it make sense to supply this value directly in the function instead of `None`? i.e.
```
def create_bullet(data, markers=None, measures=None, ranges=None,
                   subtitles=None, titles=None, orientation='h',
                   range_colors=('rgb(198, 198, 198)', 'rgb(248, 248, 248)',),
                   measure_colors=('rgb(31, 119, 280)', 'rgb(176, 196, 221)', ),
                   horizontal_spacing=None, vertical_spacing=None,
                   scatter_options={}, **layout_options):
```
(note that I'm using a tuple instead of a list because mutable default arguments aren't safe: http://docs.python-guide.org/en/latest/writing/gotchas/)